src/main/ffmpeg-optimizations-fixed.ts

```
@@ -0,0 +1,227 @@
+    // Fast seeking optimizations
+    '-ss_after_input', '1',
+    // Reduce overhead
+    '-copytb', '1',
+  ];
+
+  return optimizedArgs;
+}
+
+// Optimization 2: Improved progress handling with better performance (simplified)
+export function optimizedHandleProgress(
+  process: { stderr: any },
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)